swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefresh.kt

```diff
@@ -232,22 +227,27 @@ fun SwipeRefresh(
     onRefresh: () -> Unit,
     modifier: Modifier = Modifier,
     swipeEnabled: Boolean = true,
-    indicator: @Composable (SwipeRefreshState) -> Unit = { SwipeRefreshIndicator(it) },
+    refreshTriggerDistance: Dp = 80.dp,
```

Comment:
`80.dp` should maybe be a constant somewhere?

Reply:
I'm not sure, I personally don't feel the need to outline constants for default simple parameter values. That's what control click is for. 😄